.github/workflows/BuildPR.yml

```diff
@@ -8,34 +8,24 @@ on:
 
 jobs:
   build:
-    runs-on: ${{ matrix.os }}
-    strategy:
-      fail-fast: false
-      matrix:
-        xcode: [ Xcode_14.1.0, Xcode_13.3 ]
-        os: [ macos-12 ]
-        architecture: [ x86_64 ]
-#        include:
-#          - xcode: Xcode
-#            os: ARM64
-#            architecture: arm64
+    runs-on: macos-14
```

Comment:
Why do you removed the matrix ?

Btw, now you run an arm64 runner 👍, but I guess it makes sense to test on x86 as well.

Reply:
I've added back the CI matrix configuration. Note that on Github's macOS 15 image only Xcode 16 is available, so I've only specified that Xcode version right now.